PWGLF/Tasks/cascpostprocessing.cxx

```diff
@@ -422,10 +425,10 @@ struct cascpostprocessing {
       registry.fill(HIST("hDCANegToPV"), candidate.dcanegtopv());
       registry.fill(HIST("hDCAPosToPV"), candidate.dcapostopv());
       registry.fill(HIST("hDCABachToPV"), candidate.dcabachtopv());
-      registry.fill(HIST("hCascCosPA"), candidate.casccospa());
-      registry.fill(HIST("hV0CosPA"), candidate.v0cospa());
-      registry.fill(HIST("hCascRadius"), candidate.cascradius());
-      registry.fill(HIST("hV0Radius"), candidate.v0radius());
+      registry.fill(HIST("hCascCosPA"), candidate.pt(), candidate.casccospa());
+      registry.fill(HIST("hV0CosPA"), candidate.pt(), candidate.v0cospa());
+      registry.fill(HIST("hCascRadius"), candidate.pt(), candidate.cascradius());
+      registry.fill(HIST("hV0Radius"), candidate.pt(), candidate.v0radius());
```

Comment:
I think you can add the second axis (pt) for all topological variables. Maybe you can define ptAxisTopoVar with bins of 200 MeV each, and not 50 MeV each (50 bins in total from 0 to 10 GeV).